test/eventrouter.sh

```diff
@@ -75,20 +68,17 @@ else
     # utilize mux if mux pod exists
     if oc get dc/logging-mux > /dev/null 2>&1 ; then
         # MUX_CLIENT_MODE: maximal
-        oc label node --all logging-infra-fluentd- 2>&1 | artifact_out
-        os::cmd::try_until_text "oc get daemonset logging-fluentd -o jsonpath='{ .status.numberReady }'" "0" $FLUENTD_WAIT_TIME
+        stop_fluentd 2>&1 | artifact_out
         oc set env ds/logging-fluentd MUX_CLIENT_MODE=maximal 2>&1 | artifact_out
-        oc label node --all logging-infra-fluentd=true 2>&1 | artifact_out
-        os::cmd::try_until_text "oc get pods -l component=fluentd" "^logging-fluentd-.* Running " $FLUENTD_WAIT_TIME
+        start_fluentd true 2>&1 | artifact_out
+
         os::cmd::try_until_success "logs_count_is_gt $prev_event_count" $FLUENTD_WAIT_TIME
         prev_event_count=$( curl_es $esopssvc /.operations.*/_count?q=kubernetes.event.verb:* | get_count_from_json )
 
         # MUX_CLIENT_MODE: minimal
-        oc label node --all logging-infra-fluentd- 2>&1 | artifact_out
-        os::cmd::try_until_text "oc get daemonset logging-fluentd -o jsonpath='{ .status.numberReady }'" "0" $FLUENTD_WAIT_TIME
+        stop_fluentd 2>&1 | artifact_out
         oc set env ds/logging-fluentd MUX_CLIENT_MODE=minimal 2>&1 | artifact_out
-        oc label node --all logging-infra-fluentd=true 2>&1 | artifact_out
-        os::cmd::try_until_text "oc get pods -l component=fluentd" "^logging-fluentd-.* Running " $FLUENTD_WAIT_TIME
+        start_fluentd true 2>&1 | artifact_out
```

Comment:
Considering the CI test failure, we may not want to have "true" here?  (maybe line 73, too?)

Reply:
I'm still trying to figure out all of the different failures - still working my way through the json-file test errors - not sure why eventrouter works in the json-file test and fails in the journald test - in general using `start_fluentd true` should be safe, and should be the best way to start fluentd for a test

Reply:
@nhosoi I have a question about this commit https://github.com/openshift/origin-aggregated-logging/commit/6ff3498c91b53c59937864bf335ea550b573a184 - it added support for remote-syslog test - but it also changed forward-fluentd.sh so that it _does not_ test the two output case: https://github.com/openshift/origin-aggregated-logging/commit/6ff3498c91b53c59937864bf335ea550b573a184#diff-8e2d11cf561cc73246c98381a5ea43f9R240
I don't suppose you remember why that was added . . .

Reply:
I think I managed to fix it - needed to add the genid filter for the two output case

Reply:
Thanks, Rich.  Yeah, that's what I wanted to do... (adding to return OK... ):